src/config/IAppConfig.ts

```diff
@@ -26,6 +26,9 @@ export type StudyView = {
 
 export interface IServerConfig {
     app_name: string | null;
+    app_is_public_portal: boolean;
```

Comment:
@Rajat-Sirohi @inodb @Luke-Sikina this doesn't feel right to me.

Reply:
I think maybe what we should do is hide this yucky stuff behind feature toggle type thing.  So, there's a module FeatureToggles which has methods e.g. showPersistenceWarnings and we can then do whatever we want in there. For example read the url or whatnot.    If it's a real feature then we could of course add it properly to backend configuration